bin/cylc-scan

```diff
@@ -223,7 +225,8 @@ def main():
 
         if cre_name and not cre_name.match(name):
             continue
-        if cre_owner and not cre_owner.match(owner):
+        if (cre_owner is None and owner != get_user() or
+                cre_owner and not cre_owner.match(owner)):
```

Comment:
Fair enough, although I remember in the past that a certain *lint* program complaints about unnecessary `(...)` because it is clear to *them* that `and` is done before `or` just like `*` is before `+`.